assessments/0002-notary-project.md

```diff
@@ -0,0 +1,195 @@
+  
+- Is the Notary Project planning on localization/internationalization? If so, we recommend starting the site with a directory structure which supports localization (Hugo supports this very well out of the box).
+
+- Regarding versioning, does V1 lead directly to V2?
+   - If not, then V1 and V2 should be brought in as separate document sets, and V2 should be set up with a versioning process (allowing V1 to exist as a sort of archive).
+   - If so, then V1 can act as a base for the site and no separation of versions need be applied.
+
+#### Content creation processes
+
+<!-- borrowed from @celestehorgan's excellent work in 0001-contour.md -->
+- The following are ideas to consider going forward with a docs repo. As a content creator:
+   - If I write documentation, who verifies that it's technically accurate? 
+   - If I'm not a native english speaker, who can I ask for help with grammar and language review? 
+   - If I'm a trained technical writer and want to rearrange, create new, or split existing topics, who do I tag in issues/on Slack for comment?
+   - As a developer user, how can I ensure that content is accurate and up to date? Are alpha and beta features flagged? What happens when content (+ functionality) gets deprecated or changed in a major way? 
+   - Who reviews documentation PRs? 
+
+### Recommendations
+
+#### Information Architecture
+
+The assumption is that V2 is the primary project moving forward, and that a Minimum Viable Product (MVP) for a website/documentation will focus on Notary V2 users.
+- Bring the nv2 cli prototype documentation into the main documentation when conosolidating the content on the website.
+- V1 Docs will need to be reworked once an information architecture (IA) is setup for the new site. The new site IA can likely be informed by the existing V1 docs.
+
+The [Service Mesh Interface website](https://smi-spec.io/) handles its specification tab very well -- linking directly to the relevant markdown folder in GitHub, allowing website users to find the specification work where it is happening, and giving new users an opportunity to participate.
+
+
+## Contributor documentation 
+
+| Criteria                                  | 1   | 2   | 3   | 4   | 5   |
+| ---                                       | --- | --- | --- | --- | --- |
+| Communication methods documented          |     |     | ✅  |     |     |
+| Beginner friendly issue backlog           |     | ✅  |     |     |     |
+| “New contributor” getting started content | ✅  |     |     |     |     |
+| Project governance documentation          | ✅  |     |     |     |     |
+
+Criteria 
+- 1 = (Is not present or requires significant work)
+- 3 = (is present, but needs work)
+- 5 = (is executed extremely well or no improvement required) |
+
+### Comments & recommendations
```

Comment:
Avoid changing a templated document unless you have good reason to. In this case I'll let it slide.

Reply:
Now I'm confused. I don't think I changed the content here, just the formatting, same as above: https://github.com/cncf/techdocs/pull/27/files#r682954996

Reply:
I think this improves the readability enough to be a good reason to make the change.